tests/robustness/coverage/coverage_test.go

```diff
@@ -134,9 +134,24 @@ var referenceUsageOfEtcdAPI = map[string]refOp{
 	"etcdserverpb.KV/Compact": {
 		// Compaction should move to using internal Etcd mechanism
 		// Discussed in https://github.com/kubernetes/kubernetes/issues/80513
+		args: []column{
+			{name: "rev", matcher: isRevisionSet},
+			{name: "physical", matcher: boolAttrSet("is_physical")},
+		},
 	},
 	"etcdserverpb.Watch/Watch": {
 		// Not part of the contract interface (yet)
+		args: []column{
+			{name: "range_end", matcher: isRangeEndSet},
+			{name: "start_rev", matcher: intAttrSet("start_rev")},
+			{name: "prev_kv", matcher: boolAttrSet("prev_kv")},
+			{name: "fragment", matcher: boolAttrSet("fragment")},
+			{name: "progress_notify", matcher: boolAttrSet("progress_notify")},
+		},
+		keyAttrName: "key",
+		methods: []method{
+			{name: "Watch", matcher: notMatcher(keyIsEqualStr("key", "compact_rev_key"))},
```

Comment:
Please remember that in future iterations we would like to separate types of watch based on key it watches. Like watch for a single key vs namespace vs resource.